voice-interface/src/test/java/com/derpgroup/derpwizard/voice/model/VoiceMessageFactoryTest.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (C) 2015 David Phillips
+ * Copyright (C) 2015 Eric Olson
+ * Copyright (C) 2015 Rusty Gerard
+ * Copyright (C) 2015 Paul Winters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.derpgroup.derpwizard.voice.model;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Collections;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.derpgroup.derpwizard.voice.model.VoiceMessageFactory.InterfaceType;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class VoiceMessageFactoryTest {
+  @Mock com.amazon.speech.speechlet.IntentRequest mockAlexaRequest;
+
+  @Before
+  public void before() throws Exception {
+    com.amazon.speech.slu.Intent intent = com.amazon.speech.slu.Intent.builder().withName("test").withSlots(Collections.emptyMap()).build();
```

Comment:
Room for future enhancement. This test class needs to know about the implementing classes (currently `AlexaSimpleTextOutput` and `AlexaIntentInput`, but there will be more in the future). As it begins to mock out more of those classes, the chances for package collisions will increase, so I just assumed it would be inevitable and used fully qualified packages.